runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -112,6 +117,7 @@ func (d driver) Open(instanceID string, config map[string]any, client *activity.
 		return nil, err
 	}
 
+	var embed *embedClickHouse
 	// build clickhouse options
 	var opts *clickhouse.Options
```

Comment:
nit: Prefer section-level comments

```suggestion
	// build clickhouse options
	var opts *clickhouse.Options
	var embed *embedClickHouse
```